base-template.sh

```diff
@@ -4,6 +4,7 @@
 # It allows you to have a .githooks folder per-project that contains
 # its hooks to execute on various Git triggers.
 #
+# Legacy version number. Not used anymore, but old installs read it.
 # Version: 2006.062037-dd10de
```

Comment:
Can you please change this something somewhat invalid?
Maybe `Version: 2006.000000-000000` just so that it doesn't look like something we should be trying to interpret.

Reply:
The problem is, old update still read this, and the need to be sure its newer ?
Would you like something like `6666.000000-000000`

Reply:
Think we need somthing like `# Version: 9912.310000-000000`

Reply:
to make some sense